release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -473,6 +473,127 @@ The Local Storage Operator now has the ability to:
 [id="ocp-4-6-operators"]
 === Operators
 
+[id="ocp-4-6-operator-api"]
+==== Operator API now supported
+
+The Operator API introduced in {product-title} 4.5 as a Technology Preview
+feature is now supported and enabled by default. Previously, installing
+Operators using Operator Lifecycle Manager (OLM) required cluster administrators
+to be aware of multiple APIs, including CatalogSources, Subscriptions,
+ClusterServiceVersions, and InstallPlans. This single Operator API resource is a
+first step towards a more simplified experience discovering and managing the
+lifecycle of Operators in a {product-title} cluster.
+
+While previously only available in read-only mode, cluster administrators can
+now use the web console, in addition to the CLI, with this single API to
+interact with installed Operators. In addition, relevant OLM objects are now
+automatically labeled accordingly for the new Operator API for any Operators
+where the CSV is installed using a Subscription.
+
+Cluster administrators can discover and edit previously installed Operators
+using this API either using the web console or the CLI, for example using the
+`oc get operators` command.
+
+[id="ocp-4-6-operator-rm-tp"]
+===== Removing Technology Preview Operator API before cluster upgrade
+
+If you enabled the Technology Preview feature version of Operator API in
+{product-title} 4.5, you must disable it before upgrading to {product-title}
+4.6:
+
+.Procedure
+
+. Delete the Operator CRD:
++
+[source,terminal]
+----
+$ oc delete crd operators.operators.coreos.com
+----
+
+. Remove the `OperatorLifecycleManagerV2=true` feature gate from the OLM Operator.
+
+.. Edit the Deployment for the OLM Operator:
++
+[source,terminal]
+----
+$ oc -n openshift-operator-lifecycle-manager \
+    edit deployment olm-operator
+----
+
+.. Remove the following flags from the `args` section in the Deployment:
++
+[source,terminal]
+----
+...
+    spec:
+      containers:
+      - args:
+...
+        - --feature-gates <1>
+        - OperatorLifecycleManagerV2=true <1>
```

Comment:
Do we know what happens if we make OLM's Deployment managed by CVO again, but leave the flag alone? So long as the this doesn't leave the cluster un-upgradeable, then we may not need to do this (the flag still exists, it's just enabled by default).

Reply:
Unless there's a risk in modifying this, it feels like it'd be better to clean up any feature gate-related flags.